### PR TITLE
CC-1124 Add private load balancer mapping

### PR DESCRIPTION
**Pull request by NeilShadrach**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: terraform/groups/ecs-service/data.tf [hunk 1]
```diff
@@ -42,6 +42,15 @@ data "aws_lb_listener" "service_lb_listener" {
   port = 443
 }
 
+data "aws_lb" "secondary_lb" {
+  name = "${var.environment}-chs-apichgovuk-private" # test attach to private version of cidev main API
```
Comment (macreman) [inline terraform/groups/ecs-service/data.tf:46]: no need for the comment :)

Comment (NeilShadrach) [reply terraform/groups/ecs-service/data.tf]: I didn't think so but thought I'd leave it in and see what happened :)

Removed

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by macreman — APPROVED** [review]

**Pull request merged**